src/components/AttentionBox/AttentionBox.module.scss

```diff
@@ -81,3 +80,7 @@
 .titleContainerTitle {
   font-family: var(--title-font-family);
 }
+
+.paragraph {
```

Comment:
in attention box we don't want to add the default browser margin, in the default case we do want it according to Meirav

Reply:
<img width="607" alt="Screen Shot 2023-07-12 at 8 50 02" src="https://github.com/mondaycom/monday-ui-react-core/assets/72390374/a2df1ed2-c6c1-4fdf-b07a-7cd180781ee4">

Reply:
so why aren't we bypassing this already in the `Text` component level?

Reply:
text with paragraph prop by default have the margin in top and bottom of the component (according to Meirav request, for this stage) - in this case specifically it's not needed so removed by custom css

Reply:
so either what we see is not a paragraph or a paragraph shouldn't have margins at all.
The fact that we already need to override the default paragraph styles looks wrong